### DISCUSSION TITLE
Is it possible to populate custom fields on logger objects?

### DISCUSSION THREAD

**Discussion by Suryansh-2301** [jongpie/NebulaLogger#681] [Q&A] [answered]
We are creating logs from LWC and apex. We are able to capture information in the standard fields of the log and log entry objects. 
But if we create custom fields in these objects so is there any way to populate these fields from apex and LWC like any methods in the logger package? 
I am not able to find anything related to this in the documentation.

TIA

**Comment by jongpie** [comment, marked as answer]
Hi @Suryansh-2301 - great timing, this functionality is currently in development right now! I have an open PR, #664 (issue #655), to provide a way to populate custom fields with Nebula Logger's objects. It's still a work in progress, but the goal of the PR is to add this functionality for Apex. I definitely want to add the same functionality to LWC logging as well, but I'm just now exploring options for how to implement it in LWC, so that might be added later in a separate PR. (Similarly, I'd like to _also_ add the same functionality to Flow logging, but I'm planning to add it last.)

So, this functionality isn't available in the current version of Nebula Logger, but the plan is to have it available (in some form) for Apex, LWC, and Flow sometime in the upcoming weeks/months.

**Reply by Suryansh-2301** [reply]
Thanks for the update @jongpie really appreciate your work. I hope we will get this functionality sometime soon.

**Reply by jongpie** [reply]
@Suryansh-2301 this functionality for Apex has just been [released in `v4.13.14` of the unlocked package](https://github.com/jongpie/NebulaLogger/releases/tag/v4.13.14). It'll be available in the managed package release `v4.14.0` (PR #700).
